examples/tv-casting-app/android/App/app/src/main/java/com/chip/casting/app/MainActivity.java

```diff
@@ -66,9 +66,14 @@ public void handleCommissioningButtonClicked(DiscoveredNodeData commissioner) {
   }
 
   @Override
-  public void handleConnectionButtonClicked(CastingPlayer castingPlayer) {
-    Log.i(TAG, "MainActivity.handleConnectionButtonClicked() called");
-    showFragment(ConnectionExampleFragment.newInstance(castingPlayer));
+  public void handleConnectionButtonClicked(
+      CastingPlayer castingPlayer, Boolean useCommissionerGeneratedPasscode) {
+    Log.i(
+        TAG,
+        "MainActivity.handleConnectionButtonClicked() useCommissionerGeneratedPasscode: "
+            + useCommissionerGeneratedPasscode);
```

Comment:
We are already logging when we use the commissioner generated passcode flow in the ConnectionExampleFragment, can we avoid logging it here and anywhere else in the MainActivity?

Reply:
ok

Reply:
ok